tests/console/rsync.pm

```diff
@@ -24,6 +24,15 @@ sub run {
     my $self = shift;
     $self->select_serial_terminal;
     # try to install rsync if the test does not run on JeOS
+    # workaround for network issue
+    select_console("root-console");
+    enter_cmd("cd /var/lib/wicked");
+    my @files = qw(duid.xml lease-eth0-dhcp-ipv4.xml lease-eth0-dhcp-ipv6.xml); 
+    enter_cmd("for i in \@files; do echo '' > /var/lib/wicked/\@files; done");
+    enter_cmd("cat /var/lib/wicked/\@files");
+    assert_script_run('systemctl restart network');
+    assert_script_run('systemctl status network');
+
```

Comment:
This is a workaround that needs to be added to [`shutdown/cleanup_before_shutdown`](https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/4b9f2476dac47854eea7a1f51a569b6e68d97959/tests/shutdown/cleanup_before_shutdown.pm) as part of a workaround for: https://progress.opensuse.org/issues/116770

Reply:
yes, will do this later, thanks!